spec/fabricators/search_term_fabricator.rb

```diff
@@ -0,0 +1,3 @@
+Fabricator(:search_term) do
+  keyword "MyString"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/1d081ecd36c48dd47f0123e7164aa066b7a4d685/spec/fabricators/search_term_fabricator.rb#L2"></span>Prefer single-quoted strings when you don't need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>